runtime/src/test/java/com/dylibso/chicory/runtime/ModuleTest.java

```diff
@@ -185,35 +184,15 @@ public void shouldCountVowels() {
     @Test
     public void shouldRunBasicCProgram() {
         // check with: wasmtime src/test/resources/wasm/basic.c.wasm --invoke run
-        var instance = Module.builder("compiled/basic.c.wasm").build().instantiate();
+        var instance = Instance.builder(loadModule("compiled/basic.c.wasm")).build();
         var run = instance.export("run");
         var result = run.apply()[0];
         assertEquals(42, result.asInt());
     }
 
-    // @Test
```

Comment:
Why commented out?

Reply:
Thanks for the reminder!!!

Reply:
Uncommented all the commented tests in this file, and fixed everything around to have them running.